### ISSUE TITLE
Remove warning on doc compilation

### ISSUE THREAD

**Issue by bidoubiwa** [meilisearch/meilisearch-rust#373]
When running `cargo doc` a lot of warning are raised. For example

```
    |
373 |     /// Get [Document]s by batch.
    |              ^^^^^^^^ no item named `Document` in scope
    |
    = help: to escape `[` and `]` characters, add '\' before them like `\[` or `\]`

warning: unresolved link to `Document`
```

These should be fixed
